tests/commands/releases_list.rs

```diff
@@ -0,0 +1,76 @@
+        .create();
+
+    Command::cargo_bin("sentry-cli")
+        .unwrap()
+        .envs(common::get_base_env())
+        .args(vec!["releases", "list"])
+        .assert()
+        .success()
+        .stdout(
+            contains("| Released       | Version | New Events | Last Event |")
+                .and(contains(
+                    "| (unreleased)   | ok      | 0          | -          |",
+                ))
+                .and(
+                    is_match(r#"\| \d+ hours ago \| vue-1   \| 1          \| -          \|"#)
+                        .unwrap(),
+                ),
+        );
+}
+
+#[test]
+fn displays_releases_with_projects() {
+    let _server = mock("GET", "/api/0/projects/wat-org/wat-project/releases/")
+        .with_status(200)
+        .with_header("content-type", "application/json")
+        .with_body(r#"[{"dateReleased":"2020-03-19T10:11:35.128919Z","newGroups":1,"commitCount":0,"url":null,"data":{},"lastDeploy":{"name":null,"url":null,"environment":"x","dateStarted":null,"dateFinished":"2020-05-18T13:39:06.033442Z","id":"6447717"},"deployCount":1,"dateCreated":"2020-03-19T10:11:31.983994Z","lastEvent":null,"version":"vue-1","firstEvent":null,"lastCommit":null,"shortVersion":"vue-1","authors":[],"owner":null,"versionInfo":{"buildHash":null,"version":{"raw":"vue-1"},"description":"vue-1","package":null},"ref":null,"projects":[{"name":"test","platform":"javascript","slug":"test","platforms":["javascript"],"newGroups":1,"id":1861017}]},{"dateReleased":null,"newGroups":0,"commitCount":0,"url":null,"data":{},"lastDeploy":null,"deployCount":0,"dateCreated":"2020-03-16T16:16:12.655209Z","lastEvent":null,"version":"ok","firstEvent":null,"lastCommit":null,"shortVersion":"ok","authors":[],"owner":null,"versionInfo":{"buildHash":null,"version":{"raw":"ok"},"description":"ok","package":null},"ref":null,"projects":[{"name":"test","platform":"javascript","slug":"test","platforms":["javascript"],"newGroups":0,"id":1861017}]}]"#)
```

Comment:
is this the same as above? maybe dedupe this into a `static` or `const`?

Reply:
Most of the responses are the same (not all), and I wrote them as consts at first (see initial commits), but I unified it to use separate strings for now to make it self-contained and easier to move to `insta` crate later on.
It's also easier to just use the cli to get the output and copy it here directly.